src/kvm-host.c

```diff
@@ -0,0 +1,27 @@
+#if !defined(__x86_64__) || !defined(__linux__)
```

Comment:
The `#if` guard should exist in file `src/vm.c` which is the real KVM implementation.